src/views/LogoutSuccess.vue

```diff
@@ -19,12 +19,27 @@ import KeyguardClient from '@nimiq/keyguard-client';
 @Component({components: {Loader, SmallPage}})
 export default class LogoutSuccess extends Vue {
     @Static private request!: ParsedLogoutRequest;
-    @State private keyguardResult!: KeyguardClient.SimpleResult;
+    @State private keyguardResult!: KeyguardClient.RemoveKeyResult;
 
     public mounted() {
         WalletStore.Instance.remove(this.request.walletId);
 
-        setTimeout(() => this.$rpc.resolve(this.keyguardResult), Loader.SUCCESS_REDIRECT_DELAY);
+        let result: Logout;
+        if (this.keyguardResult.success === true) {
+            result = {
+                loggedOut: true,
+            };
+        } else {
+            result = {
+                loggedOut: false,
+                exported: {
+                    file: this.keyguardResult.file,
+                    words: this.keyguardResult.words,
```

Comment:
Since the Logout has no UI in the AM when starting the request, all CANCEL or rejects of the Keyguard request are going straight back to the caller, without hitting this component. This component `LogoutSuccess` is only executed when the request was "successful". 

But as I now found (and currently discussing with @nibhar and @Bettelstab, there might actually be _no_ way for us to get this kind of information anyway, since there is no way in `remove-key` to cancel the logout while returning a `result` - only by `reject` can a logout be cancelled, in which case this component does not run.